README.md

```diff
@@ -1,35 +1,53 @@
-RIOT Release Specs
-==================
+RIOT Release Specifications
+===========================
+
+This repository contains the criteria for a release, specifically the release
+acceptance tests.
+
+The tests are performed repeatedly on a series of release candidates, with
+iterative bugfixing. The testing for each release candidate is tracked on an
+issue in this repository. All tests passing is the goal, but if this is not
+feasible in any one release cycle due to the number of known bugs, a judgement
+call must be made by the engineers as to what is satisfactory. The known issues
+are then recorded in release-notes.txt in the main RIOT repository.
+
+For further details of the release process, see the [release management wiki
+page](https://github.com/RIOT-OS/RIOT/wiki/%5Bdraft%5D-Managing-a-Release).
+
+The scope, and automation, of the tests should be being improved continuously.
+
+Test summary
+------------
+
+The tests are listed below. Each "test" consists of a number of "tasks".
+
+| Test name                     | Description                                                      |
+|-------------------------------|------------------------------------------------------------------|
+| 01-ci                         | Compile, and run unit tests, on a range of development platforms |
+| 02-tests                      | Run other automatable tests on a selection of boards             |
+| 03-single-hop-ipv6-icmp       | Single hop IPv6 ping tests between native instances              |
+| 04-single-hop-6lowpan-icmp    | Single-hop 6LoWPAN ping tests between nodes                      |
+| 05-single-hop-route           | Static routing of ICMPv6 packets over a single hop               |
+| 06-single-hop-udp             | Single-hop UDP connectivity over IPv6                            |
+| 07-multi-hop                  | Multi-hop connectivity over IPv6                                 |
+| 08-interop                    | Interoperability of gnrc with different implementations          |
+| 09-coap                       | Send and receive of typical CoAP messages                        |
+| 10-icmpv6-error               | Single-hop ICMP connectivity over IPv6 using native              |
+| 11-lorawan                    | LoRaWAN networking                                               |
+| 99-compile-and-test-one-board | Extension - Test 02 for as many extra boards as possible         |
+
+Currently, tests 01, 02 and 09 are mostly automated. There are also some
+automation scripts [here](https://github.com/RIOT-OS/Release-Specs/pull/79) for
```

Comment:
We shouldn't refer to a PR in master. This would need rework, when #79 or an alternative is merged.

Reply:
I've removed this paragraph, all comments on future work I guess is best kept to issues and open PRs then, rather than merged documentation?

Reply:
Depends. We can of course state goals in this document. We also can say, that scripts *do* exist. But we shouldn't refer to PRs like this, as a PR is always more or less WIP until merged (especially the one referred).

Reply:
Ok, so a note saying that automation is in development within this repository, and also something to the effect that until merged, scripts should be used carefully due to issues with syncing against the specs etc? (I would have found this useful information a few weeks ago tbh)

Reply:
:+1: sounds good

Reply:
Done